website/docs/faqs/install-pip-best-practices.md

```diff
@@ -0,0 +1,18 @@
+---
+title: What are best practices for installing dbt with pip?
+---
+
+**First,** we recommend [python virtual environments](https://docs.python-guide.org/dev/virtualenvs/) to namespace `pip` modules. Here's an example setup:
```

Comment:
@VersusFacit helped @jasnonaz and I set up our pip installs and it almost killed us. We're planning to do a wider writeup of it sometime soon https://github.com/dbt-labs/devhub-writing/issues/42. This seems reasonable for the mo though!